conans/test/model/options_test.py

```diff
@@ -239,6 +239,31 @@ def test_from_list(self):
         option_values = OptionsValues(self.sut.as_list())
         self.assertEqual(option_values.dumps(), self.sut.dumps())
 
+    def test_consistency(self):
+        def _checkEqual(hs1, hs2, hs3):
+            opt_values1 = OptionsValues(hs1)
+            opt_values2 = OptionsValues(hs2)
+            opt_values3 = OptionsValues(hs3)
+
+            self.assertEqual(opt_values1.dumps(), opt_values2.dumps())
+            self.assertEqual(opt_values1.dumps(), opt_values3.dumps())
+
+        # Check that all possible input options give the same result
+        _checkEqual([('opt', 3)],       [('opt', '3'), ],       ('opt=3', ))
```

Comment:
Doesn't complain the pep8/linter?

Reply:
Because of the _tabulated_ format? Pycharm doesn't tell me anything... and I think I have almost all Inspections for Python checked.